### ISSUE TITLE
Is project alive?

### ISSUE THREAD

**Issue by jorpilo** [marta-file-manager/marta-issues#1035]
Latest release is Marta 0.8.1 from 2021. Is this project still alive?

**Comment by denisrasulev** [comment]
It looks like the developer is here (he opened a new issue in December 2023), but there's been virtually nothing done on the project since 2021. The number of issues is only increasing. So I'd say the project is rather abandoned.... :(

**Comment by bland328** [comment]
I've just discovered Marta, and _so_ wish it were still active. I'll cross my fingers that @yanex either returns to it, or might consider open sourcing the project at some point, if the project is abandoned altogether.

**Comment by bimover** [comment]
This is a truly impressive app. It would be unfortunate if it were to be neglected and abandoned. It would serve as a great project to enhance my Swift skills if it were open source.

**Comment by pax** [comment]
This would be the second my preferred file manager that was abandoned, after [DIsk Order](https://web.archive.org/web/20180718120136/http://likemac.ru/english/). 

Yan is still around, on x [@yanewm](https://twitter.com/yanewm)

**Comment by soomtong** [comment]
Yan is there; he is working on. 🎉  https://x.com/yanewm/status/1776698997329240316
> The project was on hold for some time, but now I returned. Currently, I'm wrapping up a massive refactoring of path processing that fixes various problems with symbolic link handling. Hope the new version won't take long to wait.

**Comment by Foxpit** [comment]
Please open source, let the community help.

**Comment by Offbeatmammal** [comment]
kudos for the developer for the effort this must have taken. hopefully with things like [this article](https://lifehacker.com/tech/marta-macos-finder-alternative) he can get some more support (and maybe some other devs who can help with the workload) and keep it up to date

**Comment by n1lech** [comment]
> Please open source, let the community help.

Absolutely this!

**Comment by jhult** [comment]
It is alive!

https://marta.sh/blog/back-on-track/

**Comment by Gonkers** [comment]
I just found this project today and so far it’s great. I am glad to hear it is not abandoned.

**Comment by dmt-hub** [comment]
Great project indeed! Thanks! @yanex

**Comment by soomtong** [comment]
we can close this issue :)

**Comment by ichoosetoaccept** [comment]
> It is alive!
> 
> https://marta.sh/blog/back-on-track/

Yeah it's new year's eve tomorrow and nothing has happened since that blog post. I guess Marta is dead again :(

**Comment by spenat28** [comment]
Citation from "back on track"

> I’m committed to continuing the development of Marta

... well ... it lasted exactly 0 days after.

If you made it open source at this stage, it may live ...

**Comment by reslear** [comment]
@spenat28 agree

**Comment by yanex** [comment]
After a few local improvements, I came to the conclusion that gradual evolution won't bring Marta to where I want it to be. Shortly speaking, I decided to re-build the application from the ground up, preserving only the things that I feel are already in a very good state.

The soul of Marta is to stay – it still will be the great tool for power users. However, now each feature will be made with much greater attention to detail. In addition, the application will look even more natural on macOS.

Certainly, the process is taking much more time than minor polishing, but I already adore the result. The plan is to make a series of milestones as the new version reaches the MVP.

**Comment by reslear** [comment]
@yanex nice, can you share your progress? At least with screenshots :)

**Comment by JosephTLyons** [comment]
I never thought the old one had any serious deficiencies—aesthetically pleasing, configurable, snappy.
I'm am though super interested in seeing the new version.

**Comment by spenat28** [comment]
> I never thought the old one had any serious deficiencies—aesthetically pleasing, configurable, snappy. I'm am though super interested in seeing the new version.

It will never happen ...

**Comment by yanex** [comment]
@spenat28 That's your right to think so.

By the way, I've just refactored the asynchronous operation queue with a few real benefits:
- Implementations of complex batch operations such as file copying became drastically easier;
- Several file operations working with different devices can now run concurrently. Previously, operations ran strictly one by one;
- It will be possible to implement custom operations in the re-appearing Swift API.

I agree that the refactoring takes quite some time – honestly, much more than I would like it to take. Unfortunately, I cannot afford working on Marta full-time. Still, my goal isn't to ship something half-baked as quickest as possible. I truly believe that a serious foundation I'm now building will result in a more robust, performant and a functional tool.
